homeassistant/components/keba/binary_sensor.py

```diff
@@ -22,22 +22,29 @@
     keba = hass.data[DOMAIN]
 
     sensors = [
-        KebaBinarySensor(keba, "Online", "Status", DEVICE_CLASS_CONNECTIVITY),
-        KebaBinarySensor(keba, "Plug", "Plug", DEVICE_CLASS_PLUG),
-        KebaBinarySensor(keba, "State", "Charging State", DEVICE_CLASS_POWER),
-        KebaBinarySensor(keba, "Tmo FS", "Failsafe Mode", DEVICE_CLASS_SAFETY),
+        KebaBinarySensor(
+            keba, "Online", "Status", "device_state", DEVICE_CLASS_CONNECTIVITY
+        ),
+        KebaBinarySensor(keba, "Plug", "Plug", "plug_state", DEVICE_CLASS_PLUG),
+        KebaBinarySensor(
+            keba, "State", "Charging State", "charging_state", DEVICE_CLASS_POWER
+        ),
+        KebaBinarySensor(
+            keba, "Tmo FS", "Failsafe Mode", "failsafe_mode_state", DEVICE_CLASS_SAFETY
+        ),
     ]
     async_add_entities(sensors)
 
 
 class KebaBinarySensor(BinarySensorDevice):
     """Representation of a binary sensor of a KEBA charging station."""
 
-    def __init__(self, keba, key, name, device_class):
+    def __init__(self, keba, key, name, entity_id, device_class):
```

Comment:
`entity_id` is not a good variable name since that's a specific concept within home assistant. Please call it `entity_type` instead.

Reply:
ok, makes sense!